cmd/copilot/main.go

```diff
@@ -81,6 +81,7 @@ func buildRootCmd() *cobra.Command {
 	cmd.AddCommand(cli.BuildSvcCmd())
 	cmd.AddCommand(cli.BuildJobCmd())
 	cmd.AddCommand(cli.BuildTaskCmd())
+	cmd.AddCommand(cli.BuildLocalRunCmd())
```

Comment:
Should we add the command only after the implementation is finished, or after the next release?